packages/web/test/stacktrace.test.ts

```diff
@@ -0,0 +1,223 @@
+/*
+Copyright 2024 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as assert from 'assert';
```

Comment:
need to run a `npm run lint:fix` on the files

([eslint output](https://github.com/signalfx/splunk-otel-js-web/actions/runs/11133207499/job/30958680953?pr=852))